src/openlcb/Bootloader.hxx

```diff
@@ -43,7 +43,7 @@
 #include <string.h>
 #include <unistd.h>
 
-#ifdef ESP32
+#ifdef ESP_PLATFORM
 #include "bootloader_hal.h"
 #else
 #include "freertos/bootloader_hal.h"
```

Comment:
I think we should modify libify.sh to create the freertos directory so we can consolidate these include paths, I think we had discussed this but haven't implemented a fix yet

Reply:
Filed #778